api/server.js

```diff
@@ -20,18 +20,22 @@ manifestPromise.then(manifest => {
     server.auth.strategy('oidc_session', 'oidc_session');
     server.auth.strategy('email_token', 'email_token', {
       findToken: async id => {
+        try {
         let token = await bookshelf.model('email_token')
           .forge({ token: id })
           .where('expires_at', '>', bookshelf.knex.fn.now())
           .fetch();
-
         if (!token) {
           token = await bookshelf.model('user_password_reset_token')
             .forge({ token })
             .where('expires_at', '>', bookshelf.knex.fn.now())
             .fetch();
         }
         return token;
+        }
+        catch (error){
+          return error;
```

Comment:
Returning an error isn't really appropriate. You should just let it throw and let the calling function handle it